Makefile

```diff
@@ -1,8 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
This is not the correct way to set the SHELL in a Makefile. See https://www.gnu.org/software/make/manual/html_node/Choosing-the-Shell.html.

I think we should leave this as is.